projects/libsrtp/project.yaml

```diff
@@ -2,3 +2,6 @@ homepage: "https://github.com/cisco/libsrtp"
 primary_contact: "richbarn@cisco.com"
 auto_ccs:
     - "guidovranken@gmail.com"
+sanitizers:
+ - address
+ - memory
```

Comment:
Do you intentionally exclude `undefined`? If not, please add it as well :)

Reply:
When doing a test run locally it found an unsigned int shift overflow or something and I didn't want it to crash on that.

Reply:
Got it, thanks for confirming!